src/useGenres.js

```diff
@@ -0,0 +1,28 @@
+import axios from "axios";
+import { useState, useEffect } from "react";
+import { url_back, url_front_genre } from "../src/common/API/requests";
+
+const url = `${url_front_genre}${url_back}`;
+
+export const useGenres = () => {
+    const [genres, setGenres] = useState({
+        data: [],
+    });
+
+    useEffect(() => {
+        const getGenres = async () => {
+            try {
+                const response = await axios.get(url);
+                setGenres({
+                    data: response.data.genres,
+                });
+            } catch (error) {
+                console.log(error);
+            }
+        };
+
+        setTimeout(getGenres, 0);
+    }, []);
```

Comment:
The same - setTimeout is not needed if it has the value "0".

Reply:
I'm not sure if it is a bad practice, but deleting it causes issues, so I probably would've had to rewrite the function in a different way to solve it. Also I've read somewhere that setTimeout(0) schedules the callback after the shortest possible delay, so I think it works in this instance.